### PR TITLE
OSDOCS-4692: Extending the nodeport range

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4692](https://issues.redhat.com//browse/OSDOCS-4692): Extending the nodeport range; this PR gives customers information about how to extend the NodePort range for devices using MicroShift.

Version(s):
OCP/MicroShift: 4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4692

Link to docs preview:
https://53781--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-tools.html#microshift-nodeport-range-limits_microshift-configuring

QE review:
N/A -- MicroShift docs are Developer Preview
SME acceptance is required, see lgtm label

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@mangelajo PTAL and let me know if /lgtm?

**Comment by mangelajo** [comment]
/lgtm

looks great , thanks @ShaunaDiaz

File: modules/microshift-config-nodeport-limits.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+= Extending the NodePort service range
+
+The `serviceNodePortRange` setting allows the extension of the port range available to NodePort services. This option is useful when specific standard ports under the `30000-32767` need to be exposed. For example, your device needs to expose the `1883/tcp` MQ Telemetry Transport (MQTT) port on the network because some client devices cannot use a different port.
+
+[IMPORTANT]
+NodePorts can overlap with system ports, causing a malfunction of the system or {product-title}.
+
+Considerations when configuring the NodePort service ranges:
+
+* Do not create any NodePort service without an explicit `nodePort` selection. In this case the port is assigned randomly by the `kube-apiserver`.
```
Comment (snarayan-redhat) [inline modules/microshift-config-nodeport-limits.adoc:16]:
```suggestion
* Do not create any NodePort service without an explicit `nodePort` selection. In this case, the port is assigned randomly by the `kube-apiserver`.
```

File: modules/microshift-config-nodeport-limits.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+* Do not create any NodePort service for any system service port, {product-title} port, or other services you expose on your device `HostNetwork`.
```
Comment (snarayan-redhat) [inline modules/microshift-config-nodeport-limits.adoc:18]:
```suggestion
* Do not create any NodePort service for any system service port, {product-title} port, or other services that you expose on your device `HostNetwork`.
```

**Review by snarayan-redhat — COMMENTED** [review]
Great work!! Couple of nits

**Pull request merged**